package.json

```diff
@@ -98,8 +98,8 @@
     "@types/react-helmet": "^5.0.6",
     "@types/react-hot-loader": "^3.0.6",
     "@types/react-router-dom": "^4.2.7",
+    "@types/rollup": "0.51.4",
     "archiver": "^2.1.0",
-    "awesome-typescript-loader": "5.2.1",
```

Comment:
Now there's a story. sewing-kit uses awesome-typescript-loader v3.1.3 in production, and ts-loader in development due to heap allocation issues. I using using the ts-loader version sk uses is preferable to using a version of atl different to what sk uses.

Reply:
Huh, I thought it was all the same loader. Is it for performance reasons?

Reply:
ts-loader seems to blow up and fail to compile in development for large projects from what I can tell. That was however an old version of ts-loader so it might have been fixed, foundations might reevaluate their usage and see if they can consolidate on a single thing at some point when they have time.